internal/cmd/destinations.go

```diff
@@ -14,6 +14,7 @@ import (
 
```

Comment:
This is looking pretty good. Should we filter out entries from `infra list` as they will be pending and inaccessible?

Reply:
Possibly though I'm wary of hiding pending destinations as it has a similar problem as not creating the destination until it can be connected.

Reply:
I feel list they should still have a pending label in `infra list`, otherwise they will get added to the kube config before they can be accessed (and that could take a while) which is confusing to me

Reply:
For now let's avoid putting them in `infra list` and generating a kubeconfig for pending clusters. Users can always run `infra destinations list`